frameworks/g-cloud-9/questions/services/onlineTIcketingSupportAccessibility.yml

```diff
@@ -0,0 +1,28 @@
+name: "Online ticketing support accessibility"
+question: "Does your online ticketing support comply with the Web Content Accessibility Guidelines 2.0?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: radios
+options:
+    -
+        label: "AAA"
```

Comment:
This question is out-of-date. It is currently 'What accessibility standards does your online ticketing support management meet?'